test/helper/token-validation.js

```diff
@@ -20,7 +20,7 @@ function assertTokenValidationError(
   verifier.verify(id_token, nonce, function(err, result) {
     expect(err).to.be.a(error.TokenValidationError);
     expect(err.message).to.eql(message);
-    expect(result).to.not.be.ok();
+    expect(result).to.eql(null);
```

Comment:
I wonder if we want to just continue to test for falsey here than specifically null, as that's really all we care about and how most people will check for this. Thoughts?

Reply:
The reason why I used the stricter check (specifically `null`) is so errors like the ones this PR fixes won't happen again, because forgetting the second parameter would make it `undefined` and so would fail the test. I agree that people care about falsey values, but `null` is a stricter kind of falsey.